src/Symfony/Component/Form/AbstractType.php

```diff
@@ -64,8 +64,28 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
+    public function getName()
+    {
+    }
+
+    /**
+     * Returns the prefix of the template block name for this type.
+     *
+     * The block prefixes defaults to the underscored short class name with
+     * the "Type" suffix removed (e.g. "UserProfileType" => "user_profile").
+     *
+     * @return string|null The prefix of the template block name or `null` to
+     *                     use the default block prefix
+     */
+    public function getBlockPrefix()
+    {
+    }
```

Comment:
Do you not want to implement `StringUtil::fqcnToBlockPrefix` here by default? This way it would remove the `string|null` part and the code would be self-documenting.